### DISCUSSION TITLE
hi can add a site

### DISCUSSION THREAD

**Discussion by Sirk6** [DIGITALCRIMINALS/OnlyFans#808] [Ideas]
hi, is possible to add patreon too?

**Comment by jimcrapo** [comment]
you can download from patreon using [gallery-dl](https://github.com/mikf/gallery-dl).

You'd need your cookies from patreon.com - I suggest using this [extension](https://chrome.google.com/webstore/detail/get-cookiestxt/bgaddhkoddajcdgocldbbfleckgcbcid?hl=en) to grab them. Open patreon.com, login, then download the cookies.txt file the extension generates for you.


You could then do something like this:

gallery-dl --cookies path\to\cookies.txt https://url.to.patreon.account/posts -v
